<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MENUpane</class>
 <widget class="QMainWindow" name="MENUpane">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(131, 180, 213);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="MENU_add_a_teacher">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(101, 155, 179);
color: rgb(104, 76, 119);</string>
    </property>
    <property name="text">
     <string>Add a teacher</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="MENU_edit_a_teacher">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>350</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(101, 155, 179);
color: rgb(104, 76, 119);</string>
    </property>
    <property name="text">
     <string>Edit a teacher</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="MENU_remove_a_teacher">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(101, 155, 179);
color: rgb(104, 76, 119);</string>
    </property>
    <property name="text">
     <string>Remove a teacher</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>411</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(79, 83, 126);
</string>
    </property>
    <widget class="QPushButton" name="MENU_add_a_course">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>150</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(101, 155, 179);
color: rgb(104, 76, 119);</string>
     </property>
     <property name="text">
      <string>Add a course</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="MENU_remove_a_course">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>250</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(101, 155, 179);
color: rgb(104, 76, 119);</string>
     </property>
     <property name="text">
      <string>Remove a course</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="MENU_edit_a_course">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>350</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(101, 155, 179);
color: rgb(104, 76, 119);</string>
     </property>
     <property name="text">
      <string>Remove a course</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="MENU_show_schedules">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>470</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(101, 155, 179);
color: rgb(104, 76, 119);</string>
     </property>
     <property name="text">
      <string>Show schedules</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="MENU_generate_schedules">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>470</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(101, 155, 179);
color: rgb(104, 76, 119);</string>
    </property>
    <property name="text">
     <string>Generate schedules</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
